### ISSUE TITLE
Can u share your trained best model?

### ISSUE THREAD

**Issue by zephyrrr** [Zeta36/connect4-alpha-zero#4]
I have trained a lot of hours and get loss about 1.5. and can't defeat perfect online AI agent with black or white.

**Comment by Zeta36** [comment]
Indeed, you cannot defeat a perfect brute force AI player, just get a draw.

**Comment by zephyrrr** [comment]
I use http://connect4.gamesolver.org/ to test my best model, and my model can't play the best action as web AI did.

**Comment by theputernerd** [comment]
A perfect agent always wins as player 1. https://en.wikipedia.org/wiki/Connect_Four

**Comment by theputernerd** [comment]
I would be happy to push the changes with my mcts player vs AI if you would briefly give me permissions. It provides a more definitive quality of opponent which is also incremental based on the MCTS iterations. My fork is already fairly different from this so I went back and made the minimal changes to your code.

**Comment by Zeta36** [comment]
yes, @theputernerd. Please make a pull request so I can see your changes. Include also the model weights so I can check your best game :).

Regards.
